src/main/java/org/example/homework4/MinutesToYearsAndDays.java

```diff
@@ -0,0 +1,17 @@
+package org.example.homework4;
+
+public class MinutesToYearsAndDays {
+    public static void main(String[] args) {
+        long min = 1441;
+        printYearsAndDays(min);
+    }
+    private static void printYearsAndDays(long minutes) {
+        if (minutes < 1) {
+            System.out.println("Invalid value");
+            return;
+        }
+        long yy = minutes / 525600;
```

Comment:
You used the anti-pattern called "Magic numbers"
The first questions is what is 525600?
Will you remember it in a week/month/year?
It is better to use constants with clear meaningfully name or at least to split operations like this:
```
            long hours = minutes / 60;
            long days = hours / 24;
            long years = days / 365;
            long remainingDays = days % 365;
```
it is also not perfect, but at least those magic numbers are much easier to guess